NEWS.md

```diff
@@ -1,4 +1,30 @@
-## 5.7.0 - Unreleased
+## 5.8.0 - Unreleased
+
+## 5.7.0 - Released (Poppy R2 2023)
+
+The primary focus of update DTO scheme and upgraded to java 17 version 
+
+[Full Changelog](https://github.com/folio-org/mod-invoice-storage/compare/v5.6.0...v5.7.0)
+
+### Breaking changes
+* ([MODINVOICE-466](https://issues.folio.org/browse/MODINVOICE-466)) - Update DTO schema with adding fiscalYearId to PUT payload
+
+### New APIs versions
+* Requires `PUT v5.7.0` extra fiscalYearId in DTO
+
+### Stories
+* ([MODINVOSTO-166](https://issues.folio.org/browse/MODINVOSTO-166)) - Update to Java 17 mod-invoice-storage 
+* ([MODINVOSTO-162](https://issues.folio.org/browse/MODINVOSTO-162)) - Create invoice line numbers based on new invoice field
+* ([MODINVOICE-455](https://issues.folio.org/browse/MODINVOICE-455)) - Update models schema with new fields for ftp configuration
+* ([MODINVOSTO-154](https://issues.folio.org/browse/MODINVOSTO-154)) - Update dependent raml-util
+* ([MODINVOSTO-144](https://issues.folio.org/browse/MODINVOSTO-144)) - Update DTO schema with adding fiscalYearId to PUT payload
+* ([FOLIO-3678](https://issues.folio.org/browse/FOLIO-3678)) - Use GitHub Workflows api-lint and api-schema-lint and api-doc
+
+### Bug fixes
+* ([MODORDSTOR-343](https://issues.folio.org/browse/MODORDSTOR-343)) - Duplicated PO lines have the same numbers related to newly created duplicated order
+
+### Dependencies
```

Comment:
I think we can delete this and also Breaking changes.

Reply:
it can be useful info, whay we need to delete it?

Reply:
![image](https://github.com/folio-org/mod-invoice-storage/assets/113523904/4f60bd6d-e647-41c8-9c29-3db892c6fcab)
other modules used this style, may be we consider to add extra sections